packages/medusa/src/api/auth/utils/validate-token.ts

```diff
@@ -21,7 +21,7 @@ export const validateToken = () => {
     const req_ = req as AuthenticatedMedusaRequest
 
     if (!token) {
-      return next()
+      return res.status(401).json({ message: "Invalid token" })
```

Comment:
I am just not sure if this stops the execution flow or not, cause IIRC you need to do `next(new Error(''))` to return with an error from a middleware

Reply:
@sradevski, I believe both works, as validated by the tests, but I can change to use the `new Error` to be consistent with other middleware 👍

Reply:
I would also go with next(new MedusaError) 👌